stdlib/test/os/test_atomic.mojo

```diff
@@ -87,6 +87,44 @@ fn test_atomic_floating_poInt__():
     print(atom.value)
 
 
+# CHECK-LABEL: test_atomic_move_constructor
```

Comment:
**Suggestion** I saw your comment in the commit message, but it's fine to use `assert_equal` and friends even in a test file that pipes its output to `FileCheck`.  So, I'd recommend not using `FileCheck` here and just using `assert_equal` and such from the `testing` module if you don't mind.  

This isn't blocking this PR, but would be a good habit to not create more `FileCheck` usages that we want to remove soon.

Reply:
Sure I have updated to use assert_equal